### PR TITLE
Jcs/mjml loop

### PR DESCRIPTION
**Pull request by jSylvestre**
~https://mailtrap.io/inboxes/1276099/messages/2175325360~
https://mailtrap.io/inboxes/1276099/messages/2175420960

### CODE REVIEW

**Review by sprucely — APPROVED** [review]

File: Harvest.Email/Models/AccountPendingAprovalModel.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            ProjectEnd          = "@Model.ProjectEnd";
+            ButtonUrl           = "@Model.ButtonUrl";
+            var sb = new StringBuilder();
+            sb.AppendLine("@foreach (var item in Model.AccountsList){");
+            sb.AppendLine("<tr>");
+            sb.AppendLine("<td style=\"padding: 0 15px 0 0;\">@item.Account</td>");
+            sb.AppendLine("<td style=\"padding: 0 15px;\">@item.Name</td>");
+            sb.AppendLine("<td style=\"padding: 0 0 0 15px;\">@item.Percent</td>");
+            sb.AppendLine("</tr>");
+            sb.AppendLine("}");
```
[the remark this answers is not in the document]
Comment (jSylvestre) [reply Harvest.Email/Models/AccountPendingAprovalModel.cs:38]: Ugg, I hate that you can't just paste in text

[the remark this answers is not in the document]
Comment (srkirkland) [reply Harvest.Email/Models/AccountPendingAprovalModel.cs:38]: why are we doing the foreach inside of a string and not inside of the mjml template like we do in payments?  I don't like loosing type support this way among other things

[the remark this answers is not in the document]
Comment (jSylvestre) [reply Harvest.Email/Models/AccountPendingAprovalModel.cs:38]: @srkirkland Payments translates the mjml each time with John's code.
We use a static html view

[the remark this answers is not in the document]
Comment (jSylvestre) [reply Harvest.Email/Models/AccountPendingAprovalModel.cs:38]: That just generates the mjml without removing the @foreach code

[the remark this answers is not in the document]
Comment (srkirkland) [reply Harvest.Email/Models/AccountPendingAprovalModel.cs:38]: Sure I understand we are generating it differently but i don't see why that requires that you move the c# code into a string

**Review by srkirkland — APPROVED** [review]

**Pull request merged**